doc/nrf/protocols/bt/bt_stack_arch.rst

```diff
@@ -26,13 +30,22 @@ Key features required for Bluetooth Direction Finding support are also part of t
 Feature support for PHY features depend on specific SoCs.
 Please refer to the Bluetooth feature support table on the :ref:`software_maturity` page.
 
+.. _ug_ble_controller:
+
 Controller
 **********
 
-The :ref:`nrfxlib:softdevice_controller` is the supported Bluetooth controller in the |NCS|.
-Application examples in the SDK use the SoftDevice Controller as the default controller, and it is the controller used for release testing of the SDK.
+When you develop a Bluetooth® Low Energy (LE) application, you must include a Bluetooth® LE Controller.
```

Comment:
I suggest removing this sentence, the need of a controller is obvious from the architecture. The one following is a better introduction. Choices are described later and justly so.